client/jetpack-cloud/sections/agency-dashboard/sites-overview/sites-dataviews/index.tsx

```diff
@@ -98,11 +99,19 @@ const SitesDataViews = ( {
 				id: 'site',
 				header: (
 					<>
-						<SiteSort isSortable={ true } columnKey="site">
-							<span className="sites-dataview__site-header">
-								{ translate( 'Site' ).toUpperCase() }
-							</span>
-						</SiteSort>
+						{ isA4AEnabled ? (
+							<A4ASiteSort isSortable={ true } columnKey="site">
+								<span className="sites-dataview__site-header">
+									{ translate( 'Site' ).toUpperCase() }
+								</span>
+							</A4ASiteSort>
+						) : (
+							<SiteSort isSortable={ true } columnKey="site">
+								<span className="sites-dataview__site-header">
+									{ translate( 'Site' ).toUpperCase() }
+								</span>
+							</SiteSort>
+						) }
```

Comment:
Probably this won't be needed since we are going to move SitesDataViews to A4A.

Reply:
It will depend on this PR: https://github.com/Automattic/wp-calypso/pull/88725

Reply:
Issue to track it and update our imports once that PR is merged: [Update our imports to the SitesDataViews component](https://github.com/Automattic/automattic-for-agencies-dev/issues/208)